docs/content/en/contribute/branching_model.md

```diff
@@ -0,0 +1,359 @@
+---
+title: "Branching model for the K8ssandra project"
+linkTitle: "Branching model"
+weight: 10
+description: How to branch and release K8ssandra.
+---
+
+
+
+# Branching model for the K8ssandra project
+
+
+# Purpose
+
+The current document aims at proposing a branching model for the K8ssandra project and at describing the following:
+
+*   The types of branches we should/can use
+*   The lifecycle of feature branches
+*   The process of merging feature branches into the main branch
+*   How to cut releases
+*   How to issue fixes on past releases and merge such patches back to the main branch
+
+The branching model should allow publishing patches for past releases without bringing instability by including refactorings or new features that weren’t part of the initial release.
+
+
+# Branches
+
+
+![Base branching model](images/base_branching_model.jpg "Base branching model")
+
+
+## main
+
+The main branch contains the next release and all PRs adding new features or fixes for the upcoming major/minor release should be merged in it.
+
+**This branch is protected against force push to prevent rewriting its history.**
+
+Direct pushes to main should be avoided as much as possible to allow peer review through the PR process. This also allows unit/integration tests to run before a commit is pushed to **main** as we cannot rewrite history.
+
+
+## &lt;feature>
```

Comment:
Should this use `&rt;`? For the closing brace?

```suggestion
## &lt;feature&rt;
```

Reply:
For the record &rt; isn't a thing... &gt; is. I've made this change in the most recent commit.